src/auth/auth.controller.ts

```diff
@@ -75,4 +92,43 @@ export class AuthController {
       changePasswordRequest.newPassword,
     );
   }
+
+  @HttpCode(200)
+  @Get('activate-account')
+  async activateAccount(@Query('token') token: string): Promise<void> {
+    const tokenData = await this.authService.verifyAccountActivationToken(
+      token,
+    );
+    await this.authService.activateAccount(tokenData.id);
+  }
+
+  @ApiOperation({ summary: 'Sends link to resets password of the user' })
+  @Get('reset-password-email')
```

Comment:
hmm maybe something like `request-password-reset`? Or `reset-password`, and the next one should be then `change-password`?